lib/stripe/util.rb

```diff
@@ -74,7 +74,7 @@ def self.custom_method(resource, target, name, http_verb, http_path)
                 "id should be a string representing the ID of an API resource"
         end
 
-        url = "#{resource.resource_url}/"\
+        url = "#{target.resource_url}/"\
```

Comment:
How did you conclude this change is safe? Is `target.resource_url` always equal to `resource.resource_url`?

Reply:
The target can be one of two things:

1. The resource class
2. The test helper class

The resource class already has the `resource_url` method and this PR adds one to the `APIResourceTestHelpers` base type.